stage1/init/init.go

```diff
@@ -468,6 +469,10 @@ func getArgsEnv(p *stage1commontypes.Pod, flavor string, canMachinedRegister boo
 		env = append(env, "SYSTEMD_LOG_LEVEL=err") // silence log_warning too
 	}
 
+	if hostIPC {
```

Comment:
I'd prefer this to be called `parentIPC`

Reply:
ok, I will update this.

Reply:
updated